privacy-policy.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Privacy Policy
+---
+
+If you require any more information or have any questions about our privacy policy, please feel free to [contact us by email][ContactInfo].
```

Comment:
@jnimety @lourinaldi @ZachBeta we should review this text

Reply:
If this copy is used there should be a section concerning the Meetup.com API and our use of it.

Reply:
Agreed that we should review the text. @sukima, I'm guessing this is a template? What's the source?

Reply:
@danbernier I copy/pasta it from my personal static blog site. I think I google searched for some template. But it looks like I didn't log the original source when I made mine.

The content here was intended to be a placeholder or simply to get things started.